tapir/shifts/models.py

```diff
@@ -627,6 +627,14 @@ def get_required_capabilities_display(self):
             [str(SHIFT_USER_CAPABILITY_CHOICES[c]) for c in self.required_capabilities]
         )
 
+    def get_required_capabilities_dict(self):
+        """
+        returns required capabilites as dictionary with corresponding translatiom
+        """
+        return {
+            c: _(SHIFT_USER_CAPABILITY_CHOICES[c]) for c in self.required_capabilities
```

Comment:
I would always avoid single letter names. If it's repeated a lot and makes a block very long, shortening can be fine, but here I would definitely use `capability` instead of `c`.

Reply:
I just copied your code from `get_required_capabilities_display` but changed that now :yum: